src/components/orders/OrderDetails/index.tsx

```diff
@@ -88,40 +88,45 @@ const tabItems = (
   const isLoadingForTheFirstTime = isOrderLoading && !areTokensLoaded
   const filledPercentage = order?.filledPercentage && formatPercentage(order.filledPercentage)
 
+  // Only set txHash for fillOrKill orders, if any
+  // Partially fillable order will have a tab only for the trades
```

Comment:
I think @elena-zh suggested showing the TXHash for Partials as long as the trades.length === 1

- So partials with single execution will include the TX and show it there, which is handy
- You will get the TX hash on the first execution, which might be handy too (although, i would also be ok not showing in this case)

Should we show in the Tx Hash when there's multiple fills just a link: "View order fills" or similar?

Reply:
Good to have, will address in another PR

Reply:
Actually, addressed in this PR